sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/CaseService.java

```diff
@@ -38,7 +38,7 @@ public Case buildGeneratedCase() {
         .dateOfReport(LocalDate.now())
         .externalId(UUID.randomUUID().toString())
         .disease("COVID-19")
-        .responsibleRegion("Voreingestellte Bundesländer")
+        .responsibleRegion("Voreingestellte")
```

Comment:
if you're facing issues with this, you can use: Voreingestellte Bundesl\u00E4nder  
https://itpro.cz/juniconv/ - this might help

Reply:
Good suggestion for future issues.